### ISSUE TITLE
Blacklisting Toggle for Coins and Collectibles

### ISSUE THREAD

**Issue by arielman87** [0xsequence/public-roadmap#11]
### TL;DR:
Wallet's publlic address are constantly being "bombarded" with airdrops. Most of the times these are unwanted and possibly a scam/phishing attempt. The blacklist toggle will allow devs and users to hide any token ERC-20 or collectibles ERC-721 ERC-1155 from a toggle button in settings.

### How will it look like:
<img width="600" alt="Screen Shot 2022-07-15 at 14 02 57" src="https://user-images.githubusercontent.com/97196974/179277433-74a591c2-6ecd-41ae-9c85-45a452a05c13.jpeg">

### Official Channels:
**Twitter**: https://twitter.com/0xsequence 
**Discord**: https://discord.gg/sequence 
**Reddit**: https://reddit.com/r/0xSequence/
